basculehttp/constructor.go

```diff
@@ -80,7 +113,7 @@ func (c *constructor) decorate(next http.Handler) http.Handler {
 				Authorization: key,
 				Token:         token,
 				Request: bascule.Request{
```

Comment:
Unfortunately, this is dangerous to do.  If you notice the `http.NewRequest`, it does a lot more to ensure the HTTP request object is initialized.  See https://golang.org/src/net/http/request.go#L812.

You can either (1) use `http.NewRequest`, passing in `u.String()` as the URL, or (2) replicate what `http.NewRequest` does.  I usually choose (1) since that is safe across upgrade of the stdlib.